lib/fs.js

```diff
@@ -196,10 +196,10 @@ function Stats(
   this.ino = ino;
   this.size = size;
   this.blocks = blocks;
-  this.atime = new Date(atim_msec);
-  this.mtime = new Date(mtim_msec);
-  this.ctime = new Date(ctim_msec);
-  this.birthtime = new Date(birthtim_msec);
+  this.atime = new Date(atim_msec + 0.5);
```

Comment:
I agree, these lines no longer exist though, but the +0.5s are still around and have been merged in. I'll add the comment. With #13173 the +0.5s aren't quite as needed anymore, since the goes away if you use `mtimeMs` etc. instead of `mtime`.